tests/nat/test_dynamic_nat.py

```diff
@@ -591,6 +591,12 @@ def test_nat_clear_translations_dynamic(self, ptfhost, tbinfo, duthost, ptfadapt
     @pytest.mark.nat_dynamic
     def test_nat_interfaces_flap_dynamic(self, ptfhost, tbinfo, duthost, ptfadapter, setup_test_env,
                                          protocol_type, enable_outer_interfaces):
+        # Re-enable NAT feature to ensure that the iptables rule is clean before testing
+        duthost.command("sudo config nat feature disable")
+        duthost.command("sudo iptables -F -t nat")
+        duthost.command("sudo config nat feature enable")
+        time.sleep(1)
+
```

Comment:
Why do we need to clean iptables an re-enable NAT? It looks like workaround

Reply:
In sonic-swss/cfgmgr/natmgr.cpp, the NAT rules will not be flushed after the natmgrd restart.
So if other test case doesn't disable nat feature normally (like test_nat_dynamic_enable_disable_nat_docker), the iptables rules will have residue.

The same reason for other changes.